### ISSUE TITLE
[Feature request] Provide option to archive sticker sets

### ISSUE THREAD

**Issue by caizixian** [telegramdesktop/tdesktop#2357]

**Comment by amiremami** [comment]
I wanted to open a ticket for this feature, I think this is a really good feature. Telegram please add this 😄

**Comment by auchri** [comment]
There is no api for that

**Comment by Socialdarwinist** [comment]
Now a year after the issue has been raised, I want to mention that the mobile Telegram versions (even on Windows Phone 8) are capable of moving stickers to an archive and back, whereas Telegram Desktop already has an archive tab in its sticker management page from where it is possible to move stickers archived via the phone and it is just not possible to archive the stickers from the Telegram desktop, i.e. just a small interface element needs to be implemented.

**Comment by BoffinbraiN** [comment]
I was just looking for how to do this on tdesktop as well. Seems like a significant 'whoops' in the feature list! Guess I'll have to use the Android client for now.

**Comment by mbasaglia** [comment]
I agree with this issue. Other clients have the ability to do this, seems a bit odd the desktop version can't.

**Comment by steph9009** [comment]
any updates?

**Comment by chillcat222** [comment]
Really want to see this be added

**Comment by BoffinBrain** [comment]
Now that the obnoxious github-actions bot is going around locking threads, I guess we have to keep spamming these tickets each month to keep them open. Nice job, maintainers. :(

**Comment by Aokromes** [comment]
> Now that the obnoxious github-actions bot is going around locking threads, I guess we have to keep spamming these tickets each month to keep them open. Nice job, maintainers. :(

that bot is only locking tickets closed months ago, not opened tickets.
